solidity/random-beacon/tasks/send-eth.ts

```diff
@@ -0,0 +1,60 @@
+import { task, types } from "hardhat/config"
+
+import type { BigNumber, Signer } from "ethers"
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { TransactionResponse } from "@ethersproject/abstract-provider"
+
+// eslint-disable-next-line import/prefer-default-export
+export const TASK_SEND_ETH = "send-eth"
+
+task(TASK_SEND_ETH, "Send ether to an address")
+  .addOptionalParam(
+    "from",
+    "Address to send value from",
+    undefined,
+    types.string
+  )
+  .addParam(
+    "value",
+    'Value to transfer with unit, e.g. "0.5 ether", "100 gwei"',
```

Comment:
"Amount to trasfer ..." ?

Reply:
939c42e6a5b8a6671ea74c0e72d7885386f0ca46